lib/moab/storage_object_validator.rb

```diff
@@ -50,6 +38,21 @@ def validation_errors
       errors
     end
 
+    def self.error_code_to_messages
+      @error_code_to_messages ||=
+        {
```

Comment:
maybe a comment indicating Stanford::MOV is expecting the keys to be in ascending numerical order?

Reply:
i think that might not be needed, since the subclass does `superclass.error_code_to_messages.keys.max` to get the offset for its error codes.